### PR TITLE
Adding delay to fix race conditions in continuation tests

### PR DESCRIPTION
**Pull request by macisasandwich**

### CODE REVIEW

**Comment by bryancall** [comment]
[approve ci]

File: tests/gold_tests/continuations/openclose_h2.test.py [hunk 1]
```diff
@@ -140,6 +143,7 @@
 
 # and we receive the same number of transactions as we asked it to make
 tr = Test.AddTestRun()
+tr.DelayStart = 10
```
Comment (dragon512) [inline tests/gold_tests/continuations/openclose_h2.test.py]: This one should be an extra, not needed one

File: tests/gold_tests/continuations/openclose.test.py [hunk 2]
```diff
@@ -125,10 +128,11 @@
 
 # and we receive the same number of transactions as we asked it to make
 tr = Test.AddTestRun()
+tr.DelayStart = 10
```
Comment (dragon512) [inline tests/gold_tests/continuations/openclose.test.py]: This one should be an extra case we can remove

**Comment by dragon512** [comment]
I think we might be able to remove some other delays... We needed the delays after we pushed data through ATS. We need the delay so the "traffic_ctrl get xxx" commands had time to get updated information. Once the first one is up-to-date move the "get" calls should be fine as well.

**Pull request merged**